tests/conftest.py

```diff
@@ -11,6 +11,11 @@
 
 
 def pytest_report_header(config):
+    # Performed in this function to enable it before any test collection
```

Comment:
Is there a better function to call this in? I couldn't find any pytest documentation about the order of hooks, but this one definitely occurs before any collection.

Reply:
https://docs.pytest.org/en/6.2.x/reference.html#initialization-hooks strongly suggests to me that you want `pytest_configure`